guides/common/Makefile

```diff
@@ -60,6 +60,8 @@ serve: html
 linkchecker: html
 	linkchecker -r1 -f $(COMMON_DIR)/linkchecker.ini --check-extern "file://$(realpath $(ROOTDIR)/$(DEST_HTML))"
 
+toc: html
```

Comment:
Is this target used? I don't think it makes sense for individual guides so I'd leave it out.

Reply:
The top level task calls the individual task with the same name, so I had to leave a task called `toc` for each guide. On the other hand i have changed the individual task to noop.